src/opencl_pdf_fmt_plug.c

```diff
@@ -0,0 +1,509 @@
+/*
+ * This software is Copyright (c) 2024 magnum and it is hereby released to the
+ * general public under the following terms:  Redistribution and use in source
+ * and binary forms, with or without modification, are permitted.
+ */
+
+#define FORMAT_STRUCT fmt_pdf_opencl
```

Comment:
Why do you need the `FORMAT_STRUCT` macro?

Reply:
It was in whatever I used as template - probably something predating our current use/storing of `self`.

Reply:
Oh, I now see we do have this `FORMAT_STRUCT` thing in a couple of formats already. This will be the third.